### DISCUSSION TITLE
Amstrad CPC 3"

### DISCUSSION THREAD

**Discussion by pilou75** [keirf/greaseweazle#271] [General]
Hi

I recently bought a greaseweazle in order to use it with a 3" drive (internal CPC 6128 -> single sided)
It has a 26 pins connector. A special ribbon has to be made, and I found LOTS of conflicting informations on the internet.

So, just a few lines, to let you know that what is explained here (https://www.fvempel.nl/3pc.html) worked well for me. 
I checked carefully what I did before using of course and 1st attempt was successful - read and write unusual format.
Be very careful with the power supply as 5V and 12V are swapped, it's *NOT* like an IBMPC.

**Comment by Breiztiger** [comment]
can this be ok for amstrad cpc 3" floppy dump ?

disk amstrad.cpc
    cyls = 42
    heads = 1
    tracks * ibm.mfm
        secs = 9
        bps = 512
        gap3 = 84
        rate = 250
    end
end
